### ISSUE TITLE
Pretrained model with wav2vec-u for Romanian language

### ISSUE THREAD

**Issue by Moldoteck** [pytorch/fairseq#3615]
## 🚀 Feature Request
Pretrained model with wav2vec-u for Romanian language

### Motivation

There are no good/opensource models for this language. Since the last wav2vec-u method does not need to have labeled data, I think it can be pretty doable to create such a model and then let other developers to fine tune it

### Pitch

Model which transcribes Romanian audio speech into text format

### Alternatives

Since Romanian is based on latin letters, special characters could be mapped to that latin letters(like ă->a, î>i and so on)

**Comment by jimregan** [comment]
There is a [wav2vec2 model](https://huggingface.co/anton-l/wav2vec2-large-xlsr-53-romanian)

**Comment by Moldoteck** [comment]
> There is a [wav2vec2 model](https://huggingface.co/anton-l/wav2vec2-large-xlsr-53-romanian)

Yes, but the test performance is only 25% and by testing it myself with pretty simple and clear samples, it is not quite accurate. That's why I was hoping that this new wav2vec which doesn't require labeled data maybe will be able to perform a bit better

**Comment by JeromeNi** [comment]
> > There is a [wav2vec2 model](https://huggingface.co/anton-l/wav2vec2-large-xlsr-53-romanian)
> 
> Yes, but the test performance is only 25% and by testing it myself with pretty simple and clear samples, it is not quite accurate. That's why I was hoping that this new wav2vec-u which doesn't require labeled data maybe will be able to perform a bit better

To the best of my knowledge, when training the unsupervised model in the first step, the actual wav2vec2 representations were frozen. Those layers weren't updated until the second self-training stage with pseudo transcriptions from an HMM, where the HMM itself is trained on pseudo transcriptions from the unsupervised model in the first step. There is no pre-training involved, at least in the original wav2vec sense.

**Comment by alexeib** [comment]
if you already have a model which you have already fine-tuned to a semi-ok performance, you can likely significantly improve it by self-training on some corpus of similar-domain unlabeled audio data. you can use e.g. w2v -> HMM -> finetune w2v pipeline, or just w2v -> finetune w2v (the delta wont be too huge) and see what you get after a couple of rounds
